src/Native/Runtime/RHCodeMan.cpp

```diff
@@ -1366,6 +1416,8 @@ bool EECodeManager::FindNextEpilog(GCInfoHeader * pInfoHeader, UInt32 methodSize
 
 #ifdef _ARM_
 #define IS_FRAMELESS() ((pInfoHeader->GetSavedRegs() & CSR_MASK_LR) == 0)
+#elif defined(_ARM64_)
+#define IS_FRAMELESS() (throw "FIXME:ARM64", 0 == 0)
```

Comment:
I think `IS_FRAMELESS()` really ought to be an inline function instead of a macro at this point.  It'll make it easier for your implementation to just contain a `PORTABILITY_ASSERT` in it without having to go to the rarely used comma operator and an expression that evaluates to a constant.  :)